exporter/influxdbexporter/README.md

```diff
@@ -0,0 +1,141 @@
+# InfluxDB Exporter
+
+This exporter supports sending tracing, metrics, and logging data to [InfluxDB](https://www.influxdata.com/products/).
+
+Supported pipeline types: traces, metrics, logs
+
+## Configuration
+
+The following configuration options are supported:
+
+* `org` (Required): Org is the InfluxDB organization name of the destination bucket.
+* `bucket` (Required): Bucket is the InfluxDB bucket name that telemetry will be written to.
+* `protocol` (Optional): Protocol is the method used to deliver metrics to InfluxDB. Must be one of `line-protocol`, `grpc`. Defaults to `line-protocol`.
+* `line_protocol_server_url` (Required): LineProtocolServerUrl is the destination for Protocol `line-protocol`.
+* `line_protocol_auth_token`: LineProtocolAuthToken is the authentication token for Protocol `line-protocol`.
+* `timeout` (Optional): Timeout is the timeout for every attempt to send data to InfluxDB. Defaults to `5s`.
+
+
+Example:
+```yaml
+exporters:
+  influxdb:
+    org: "my-org"
+    bucket: "my-bucket"
+    protocol: "line-protocol"
+    line_protocol_server_url: "http://localhost:8086"
+    line_protocol_auth_token: "my-token"
+    timeout: 10s
+```
+
+The full list of settings exposed for this exporter are documented [here](config.go).
+
+## Schema
+
+InfluxDB/IOx is an open-source time series storage engine, capable of storing and querying large (and small) time series datasets.
+The schema used by this OpenTelemetry exporter is optimized for InfluxDB/IOx performance.
+
+### Definitions
```

Comment:
This is a nice writeup. I wonder if it should be extracted and moved elsewhere (can be somewhere in this repo) since it is not Influxdb-specific.

Reply:
I agree that it doesn't belong here. My original audience for this section was other Influx employees, not the OpenTelemetry community.

The original is copied out to a text file here. Can you suggest a specific destination for it?